ci-operator/step-registry/interop-tests/openshift-virtualization-tests/interop-tests-openshift-virtualization-tests-commands.sh

```
@@ -1,231 +1,269 @@
-    if [ -n "$cmd_yq" ]; then
-        echo "yq version: $cmd_yq"
+InstallYqIfNotExists() {
+    typeset cmdYq
+    cmdYq="$(yq --version 2>/dev/null || true)"
+    if [[ -n "${cmdYq}" ]]; then
+        echo "yq version: ${cmdYq}"
     else
-        echo "Installing yq"
         mkdir -p /tmp/bin
-        export PATH=$PATH:/tmp/bin/
+        export PATH="${PATH}:/tmp/bin/
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)